### ISSUE TITLE
Display current preferences

### ISSUE THREAD

**Issue by pneubauer** [tschak909/platoterm64#29]
There is is no user interface for viewing preferences, such as selected baud rate.  I often find myself re-entering preferences because I've forgotten what I entered previously.

I suggest an enhancement that summarizes the current preferences, like "Serial (Slot 2), 1200 baud 8N1", upon startup.

**Comment by tschak909** [comment]
Fair enough. I will implement this when I can get a chance, if someone else doesn't before I get back to it. :)

**Comment by pneubauer** [comment]
I agree this is a low priority, and I intend to contribute code.  At the moment, I'm documenting "issues" when I find them as a small way to support the project.

**Comment by tschak909** [comment]
I appreciate you doing this. :)
